content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md

```diff
@@ -202,4 +202,6 @@ verify that the pods were scheduled by the desired schedulers.
 ```shell
 kubectl get events
 ```
+You can also use a [custom scheduler configuration,](https://kubernetes.io/docs/reference/scheduling/config/#multiple-profiles) or custom container image, for the cluster's main scheduler by modifying its static pod manifest on the relevant control plane host.
```

Comment:
```suggestion
You can also use a [custom scheduler configuration](/docs/reference/scheduling/config/#multiple-profiles)
or a custom container image for the cluster's main scheduler by modifying its static pod manifest
on the relevant control plane nodes.
```

Reply:
I'd use a glossary tooltip for “static pod” as it is a term readers might not recognize. Here's
how I'd put it @vaibhav2107:
```suggestion
You can also use a
[custom scheduler configuration](/docs/reference/scheduling/config/#multiple-profiles)
or run a custom container image for the cluster's main scheduler. The usual way to make
either change is by modifying the
{{< glossary_tooltip text="static pod" term_id="static-pod" >}} manifest for the
kube-scheduler component on the relevant control plane nodes.
```